addons/website_slides_survey/controllers/main.py

```diff
@@ -8,6 +8,19 @@
 class WebsiteSlidesSurvey(http.Controller):
     @http.route(['/slides_survey/certification/search_read'], type='json', auth='user', methods=['POST'], website=True)
     def slides_certification_search_read(self, fields):
+        can_create = request.env['survey.survey'].check_access_rights('create', raise_exception=False)
         return {
             'read_results': request.env['survey.survey'].search_read([('certificate', '=', True)], fields),
+            'can_create': can_create,
```

Comment:
weird to return a can_create flag in a search_read route.
Anyway, this route should even not exist as it can be done in js using rpc on search_read method on the model, but not your fault :)

Reply:
I did it the same way as it was done for the categories:
```
    @http.route(['/slides/category/search_read'], type='json', auth='user', methods=['POST'], website=True)
    def slide_category_search_read(self, fields, domain):
        category_slide_domain = domain if domain else []
        category_slide_domain = expression.AND([category_slide_domain, [('is_category', '=', True)]])
        can_create = request.env['slide.slide'].check_access_rights('create', raise_exception=False)
        return {
            'read_results': request.env['slide.slide'].search_read(category_slide_domain, fields),
            'can_create': can_create,
        }
```
read_results are used to create the dropdown and can_create is used to allow or not to create new values directly in the dropdown.

Reply:
maybe you could take the opportunity to rename the controller (route + function), indeed, as it's now more then just getting all certif.